[Python] Port strings concept exercise from C#. (#760)
exercism/v3: languages/python/exercises/concept/strings/.docs/after.md

# After

> 

Reviewer: We've recently (as in yesterday) started to experiment with changing the `after.md` document a bit. The most important change is that the `after.md` document should now summarize what the exercise aimed to teach the student, like a recap of sorts. It can also still include references to interesting other resources like the ones you list here.

For an example of new `after.md` files, see https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/strings/.docs/after.md and https://github.com/exercism/v3/pull/767/files

It's useful to become familiar with Python's string instance methods, formatting, and literal syntax. For a deep dive on what information a string is encoding (or, "how does the computer know how to translate zeroes and ones into letters?"), this blog post is enduringly helpful: https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/

### More about Python logging

> Python has a useful builtin logging library, aptly-named `logging`. It handles cases similar

Reviewer: ```suggestion
Python has a useful built-in logging library, aptly-named `logging`. It handles cases similar
```

I like this reference!

> # After

Reviewer: ```suggestion
```

We've recently removed these headers.

In this lesson, we learned:
 - Python has a `str` type.
 - How to find items in a string.
 - How to manipulate strings to create new strings.
 - Where to find documentation about string methods.

## More about strings

> It's useful to become familiar with Python's string instance methods, formatting, and literal syntax. For a deep dive on what information a string is encoding (or, "how does the computer know how to translate zeroes and ones into letters?"), this blog post is enduringly helpful: https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/

Reviewer: Maybe use a link?

```suggestion
It's useful to become familiar with Python's string instance methods, formatting, and literal syntax. For a deep dive on what information a string is encoding (or, "how does the computer know how to translate zeroes and ones into letters?"), [this blog post is enduringly helpful](https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/).
```

### More about Python logging

> Python has a useful built-in logging library, aptly-named `logging`. It handles cases similar

Reviewer: Maybe link in the text instead of as a separate bit of text?

```suggestion
Python has a useful [built-in logging library](https://docs.python.org/3.8/library/logging.html), aptly-named `logging`. It handles cases similar
```
